.github/workflows/nightly-release.yaml

```diff
@@ -0,0 +1,43 @@
+name: Nightly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 2 * * *" # release nightly at 2AM
```

Comment:
Curious question: do you know what's the timezone of the servers that are running this?

Reply:
Time is expressed in UTC https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule